src/runtimes.js

```diff
@@ -0,0 +1,19 @@
+const path = require("path");
+const fs = require("fs");
+
+const RuntimeType = {
+  dotnet: "dotnet",
+  node: "node",
+};
+
+module.exports.RuntimeType = RuntimeType;
+
+module.exports.detectRuntime = (app_location) => {
+  const files = fs.readdirSync(app_location);
+
+  if (files.some((file) => path.extname(file) === ".csproj")) {
+    return RuntimeType.dotnet;
+  }
+
+  return RuntimeType.node;
```

Comment:
```suggestion
  if (files.some((file) => path.extname(file) === "package.json")) {
    return RuntimeType.node;
  }
  
  return RuntimeType.unknown;
```

Reply:
This is part of the changes in #10 (but this PR builds on top of it), do you want me to close #10 and just have one PR covering a few features?

Reply:
okay. let's do that. the project is still in alpha. let's go yolo ^^